src/ngraph/pass/visualize_tree.cpp

```diff
@@ -376,6 +377,10 @@ string pass::VisualizeTree::get_attributes(shared_ptr<Node> node)
             label << " " << (node->get_output_size() != 1
                                  ? string("[skipped]")
                                  : pretty_partial_shape(node->get_output_partial_shape(0)));
+            if (auto reshape = dynamic_cast<const op::Reshape*>(&(*node)))
```

Comment:
Use `auto reshape = as_type_ptr<op::Reshape>(node)`. dynamic cast has subtle problems on some platforms.

Reply:
will make the change, since node isnt passed as reference i wasnt sure as_type_ptr would work. Looks like as_type_ptr does support val and ref.